### PR TITLE
Adds some new tile types, grates. Fixes #3332

### LINKED ISSUE #3332: Missing broken floor ramp sprite
https://puu.sh/xgYM8/18ac87e7a1.png
I think it's the boken state

### PR DESCRIPTION
**Pull request by fernerr**
Thanks to Happy Fox for their spriting efforts.

This pr adds some assets that are not mapped in yet, but are to be used in future projects among other things.

 - Added a new type of linoleum, rubber carpets, dark/light metal grates. Added some proper footramp damaged states. 
 - Makes blue/green (circuit/strata) tiles craftable somehow like requested here, though via autolathe instead.
https://forums.aurorastation.org/topic/12800-please-make-the-data-strata-floor-craftable/
 - Blown up footramps will no longer result in a 'Fix me!' icon appearing.
Fixes #3332

### CODE REVIEW

**Comment by fernerr** [comment]
!review

**Comment by alsoandanswer** [comment]
Do they have proper footstep sounds?

**Comment by fernerr** [comment]
Yes.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 1]
```diff
@@ -58,14 +58,96 @@
...
+
+/obj/item/stack/tile/circuit_blue
+	name = "circuit tile"
+	singular_name = "circuit tile"
+	desc = "An advanced tile covered in various circuitry and wiring."
+	icon_state = "tile_bcircuit"
+	force = 6.0
+	matter = list(DEFAULT_WALL_MATERIAL = 937.5,"glass" = 937.5)
+	throwforce = 15.0
+	throw_speed = 5
```
Comment (NonQueueingMatt) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: That's a LOT of throwforce for something this easy to make.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 2]
```diff
@@ -58,14 +58,96 @@
...
+
+/obj/item/stack/tile/circuit_green
+	name = "circuit tile"
+	singular_name = "circuit tile"
+	desc = "An advanced tile covered in various circuitry and wiring."
+	icon_state = "tile_gcircuit"
+	force = 6.0
+	matter = list(DEFAULT_WALL_MATERIAL = 937.5,"glass" = 937.5)
+	throwforce = 15.0
+	throw_speed = 5
```
Comment (NonQueueingMatt) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: As above.

**Pull request merged**